lib/onebox/sanitize_config.rb

```diff
@@ -1,72 +0,0 @@
-# frozen_string_literal: true
-
-class Sanitize
```

Comment:
I'm curious what the context is for the removal of this.

Reply:
Actually I moved the code to `lib/onebox/discourse_onebox_sanitize_config.rb` as we were reopening the `sanitize` gem for no good reason.

Reply:
Thank you for the explanation :+1: